include/onnxruntime/core/optimizer/rewrite_rule.h

```diff
@@ -35,6 +35,17 @@ If the list of op types is left empty, that rule will be triggered for every op
 */
 class RewriteRule {
  public:
+  /**
+  @class RewriteRuleEffect
+
+  Class used to indicate the effect of rule application on a graph's node.
+  */
+  enum class RewriteRuleEffect : uint8_t {
+    kUnmodifiedGraph,    // The graph has not been modified by the rewrite rule.
```

Comment:
>kUnmodifiedGraph [](start = 4, length = 16)

'unmodified graph' could also mean that modifications were undone. Maybe kNone or something like that given this is the value for "what was the rewrite rule's effect"?

Reply:
I used the values described above (including the kNone).